scripts/resource-creation.sh

```diff
@@ -35,4 +55,5 @@ sudo pip3 uninstall crcmod
 sudo pip3 install pipenv
 sudo pip3 install --no-cache-dir -U crcmod
 
+#cd necstlab-damage-segmentation && pipenv install --skip-lock        # reed needs this
```

Comment:
is this a thing that you need to run on the remote machine or that's just the commend you need to run it locally on windows?

Reply:
removed